### PR TITLE
Create a new storage-selector component

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Create a new storage-selector component

Use for image component

[stage-2]

## Motivation and Context
Component Navigation Improvements

## How Has This Been Tested?
Tested changes locally. Will fix issues and update tests in subsequent PRs.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
Staged version is overwritten by https://github.com/Rise-Vision/rise-vision-apps/pull/2534

@ezequielc please review. Thanks!

File: src/scripts/template-editor/components/directives/dtv-component-image.js [hunk 1]
```diff
@@ -267,6 +231,25 @@ angular.module('risevision.template-editor.directives')
             }
           });
 
+          var logoDirective = angular.extend({}, baseDirective, {
+            type: 'rise-image-logo',
+            icon: 'circleStar',
+            title: 'Brand Settings',
```
Comment (ezequielc) [inline src/scripts/template-editor/components/directives/dtv-component-image.js:237]: @alex-deaconu Title should be 'Logo Settings' here. Thanks!

Comment (alex-deaconu) [reply src/scripts/template-editor/components/directives/dtv-component-image.js:237]: @ezequielc good catch, will update. Thanks!

Comment (alex-deaconu) [reply src/scripts/template-editor/components/directives/dtv-component-image.js:237]: @ezequielc fixed in the other PR: https://github.com/Rise-Vision/rise-vision-apps/pull/2535/commits/a8aef81c67573075842e7be9f9780bf7b5aa6563. Thanks.

**Review by ezequielc — APPROVED** [review]

**Pull request merged**